english/motor_oil/motor_oil.md

```diff
@@ -0,0 +1 @@
+Humans are unable to digest motor oil at this time.  We humbly suggest that you do not endeavour to ingest motor oil for the forseable future... including future lives.  Your princess is in another castle, you have failed... Good bye.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.